### DISCUSSION TITLE
Wrong email info@dataspace-connector.de to ask to unblock IP addresses

### DISCUSSION THREAD

**Discussion by ctoscano-eu** [International-Data-Spaces-Association/DataspaceConnector#119] [Q&A] [answered]
I would like to start using the Metadata Broker from the Dataspace Connector. 
According to https://international-data-spaces-association.github.io/DataspaceConnector/CommunicationGuide/v6/IdsEcosystem/Broker  I have to ask someone to unblock the IP addresses of the my machines holding instances of the Dataspace Connector by using this email  info@dataspace-connector.de
But this email doesn't exist.

thank you in advance, my best regards
César

**Comment by HeinrichPet** [comment]
Hi César, 

sorry for the late reply.
The Metadata Broker in IDS Lab is not yet available on the latest version, for this reason it is disabled since December (Log4J Vulnerability). Therefore an unblock currently makes no sense.
I tried to get you an alternative, but I haven't found one yet.
Maybe @SebastianOpriel & @mborowski511 can give you alternatives soon.

Best regards,
Heinrich

**Comment by ctoscano-eu** [comment]
Thank you Heinrich, this is not urgent. The goal was to start testing the access to the Broker. We can do it when possible.
Bets regards
César

**Comment by juliapampus** [comment, marked as answer]
The e-mail address does exist. I just tested.
